priv/repo/migrations/20211125141835_optimize_audios_uuid.exs

```diff
@@ -0,0 +1,16 @@
+# Use 36 bytes of fixed storage for storing UUIDs instead of a dynamically
+# allocated storage of `1 + 36 * 3` bytes (109 bytes) with UTF8, without
+# changing the actual UUID format.
+defmodule Ask.Repo.Migrations.OptimizeAudiosUuid do
+  use Ecto.Migration
+
+  def up do
+    execute "ALTER TABLE audios MODIFY uuid CHAR(36) CHARACTER SET ascii"
+    create index(:audios, :uuid)
+  end
+
+  def down do
+    execute "ALTER TABLE audios MODIFY uuid VARCHAR(255) CHARACTER SET utf8"
+    drop index(:audios, :uuid)
```

Comment:
Not sure if dropping the index _before_ altering the table would avoid the DB some work re: updating the index that will get dropped anyway.

Reply:
Indeed!